src/renderer/common/js/utils.js

```diff
@@ -92,18 +91,21 @@
      * @return {Boolean} - returns `true` on successfully executed file
      */
     gpii.psp.launchExecutable = function (executablePath) {
-        var fileProperties = fs.statSync(executablePath);
-
-        // Check that the file is executable
-        if (fileProperties.mode === parseInt("0100666", 8)) {
-            try {
-                child_process.exec(executablePath);
-                return true;
-            } catch (err) {
-                fluid.log(fluid.logLevel.WARN, "launchExecutable: Cannot execute - " + executablePath);
+        try {
+            var fileProperties = fs.statSync(executablePath);
+            // Check that the file is executable
+            if (fileProperties.mode === parseInt("0100666", 8)) {
+                try {
+                    child_process.exec("\"" + executablePath + "\"");
```

Comment:
yes, this really made the difference :wink:

Reply:
Yeah, my bad :-). I did knew about this, even used it for the executing the shareX command, but totally forgot to put it here... well, it works now, that's important!

And even passed all the tests, so its ready to be merged as well.

@amb26 anything you want to be changed before the merge?